.circleci/config.yml

```diff
@@ -64,6 +64,15 @@ jobs:
           command: |
             sudo pip install tox
             tox -e py39
+  py39-notrio:
+    docker:
+      - image: circleci/python:3.9
+    steps:
+      - checkout
+      - run:
+          command: |
+            sudo pip install tox
+            tox -e py39-notrio
```

Comment:
inconsistent with `tox.int` (`py39-trio` for `trio` and normal python versions for other)